src/components/InfoBanner.tsx

```diff
@@ -20,47 +25,46 @@ const InfoBanner = ({
   shouldCenter,
   shouldSpaceBetween,
   title,
+  className,
+  onClick,
   ...props
 }: InfoBannerProps) => {
   const banner = (
-    <LightMode>
+    <VStack
+      className={cn(
+        "rounded-sm p-6 sm:flex-row",
+        shouldCenter ? "max-w-[55rem]" : "max-w-full",
+        isWarning ? "bg-orange-200" : "bg-indigo-100",
```

Comment:
@nloureiro would like to have your eyes on these colors and how to migrate them to the new theme colors

Reply:
as discussed offline, we are doing:
- info => `bg-primary-low-contrast`
- warning => `bg-warning-light`